chef/data_bags/crowbar/migrate/nova/023_add_max_header_line.rb

```diff
@@ -0,0 +1,9 @@
+def upgrade ta, td, a, d
+  a['max_header_line'] = 16384
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.